docs/CONTRIBUTING.md

```diff
@@ -70,7 +70,7 @@ that's required to go with a given prefix.
 Unfortunately, these requirements can not be applied retroactively and can not
 be trivially applied to automatically imported prefixes. In some cases,
 historical prefixes can be modified to follow these requirements. For example,
-Identifiers.org's `ec-code` was renamed to `eccode` while maintaining `ec-code`
+Identifiers.org's `ec-code` was renamed to `ec` while maintaining `ec-code`
```

Comment:
this is a bit misleading. The point of this paragraph is to explain the normalization that happens to external prefix registries' prefixes. Maybe we need to find a different example from identifeirs.org

Reply:
I just ran the following script and didn't find any other good examples to illustrate prefix normalization, so I think it's okay to just leave it since ec-code was the only good example here and there isn't a more general thing going on

```python
import bioregistry

if __name__ == '__main__':
    for resource in bioregistry.resources():
        miriam = resource.get_miriam_prefix()
        if not miriam or not "-" in miriam:
            continue
        if miriam.replace("-", "") == resource.prefix:
            print(resource.prefix)
```

Reply:
Yes, this change makes the example a bit awkward but it is probably okay to keep